test/unit/lib/plugins/aws/lib/monitor-stack.test.js

```diff
@@ -669,26 +676,28 @@ describe('monitorStack', () => {
       describeStackEventsStub.onCall(2).resolves(deleteItemFailedEvent);
       describeStackEventsStub.onCall(3).resolves(deleteFailedEvent);
 
-      return awsPlugin.monitorStack('delete', cfDataMock, { frequency: 10 }).catch((e) => {
-        let errorMessage = 'An error occurred: ';
-        errorMessage += 'mochaLambda - You are not authorized to perform this operation.';
-        expect(e.name).to.be.equal('ServerlessError');
-        expect(e.message).to.be.equal(errorMessage);
-        // callCount is 2 because Serverless will immediately exits and shows the error
-        expect(describeStackEventsStub.callCount).to.be.equal(3);
-        expect(
-          describeStackEventsStub.calledWithExactly('CloudFormation', 'describeStackEvents', {
-            StackName: cfDataMock.StackId,
-          })
-        ).to.be.equal(true);
-        awsPlugin.provider.request.restore();
-      });
+      await expect(
+        awsPlugin.monitorStack('delete', cfDataMock, { frequency: 10 })
+      ).to.eventually.be.rejectedWith(
+        ServerlessError,
+        'An error occurred: mochaLambda - You are not authorized to perform this operation.'
+      );
+
+      // TODO: why this is 3?
```

Comment:
Sorry, I think I forgot post this comment. It's about above line, let's remove it